scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -414,86 +406,86 @@ head( rowData(merged_sce) )
 
-```{r merged_sce view pca}
+```{r merged sce view pca}
 reducedDimNames(merged_sce)
 ```
 
-These represent the original dimension reductions that were performed on _each individual library_, but we actually need to calculate PCA (and UMAP for visualization) from the merged object directly.
+These represent the original dimension reductions that were performed on _each individual SCE_ before merging, but we actually need to calculate PCA (and UMAP for visualization) from the merged object directly.
 
-Why can't we use the library-specific PCA? 
-Part of the PCA calculation itself involves scaling the raw data to center the mean.
-When this is performed on each library individually, all libraries are separately centered, which leads to the data "overlapping" in space.
-To see this, let's look at the UMAP for individual libraries:
+Why can't we use the sample-specific PCA matrix? 
+Part of PCA calculation itself involves scaling the raw data to center the mean.
+When this is performed on each sample individually, all samples are separately centered, which leads to the data "overlapping" in space.
+To see this, let's look at the UMAP for individual samples:
 
 ```{r individual UMAPs}
-# UMAPs scaled separately when calculated from individual libraries:
+# UMAPs scaled separately when calculated from individual samples:
 scater::plotReducedDim(merged_sce,
                        dimred = "UMAP",
-                       # Color points based on the `library` variable we added during merging
-                       colour_by = "library",
+                       # Color points based on the `sample` variable we added during merging
+                       colour_by = "sample",
                        # Include some point styling to help us see the points
                        point_size = 0.5,
                        point_alpha = 0.2) +
   # Modify the legend key so its points are larger and easier to see
   guides(colour = guide_legend(override.aes = list(size = 3, alpha = 1))) +
   # Add a plot title
-  ggtitle("UMAP calculated on each library separately")
+  ggtitle("UMAP calculated on each sample separately")
 ```
 
 
-As we see in this UMAP, all libraries are centered at zero and all overlapping!
+As we see in this UMAP, all samples are centered at zero and all overlapping!
 This visual artifact can give the _incorrect impression_ that data is integrated - to be clear, this data is NOT integrated!
 
-For input to integration, we'll want the PCA scaling to consider normalized gene expression values from all libraries simultaneously, while also taking into consideration batch differences, so we'll need to recalculate PCA (and UMAP for visualization). 
+For input to integration, we'll want the PCA scaling to consider normalized gene expression values from all samples simultaneously, while also taking into consideration batch differences.
```

Comment:
I think the final clause is confusing here: people might interpret it to mean that there is batch correction happening here, but it is mostly a weighting for the PCA itself (the gene selection is more of a traditional batch correction)... and we will cover it when we get there.
```suggestion
For input to integration, we'll want the PCA scaling to consider normalized gene expression values from all samples simultaneously.
```